api/repo_test.go

```diff
@@ -64,12 +64,6 @@ func (suite *RepoSuite) TestCreateOperation() {
 	user, err := models.Users(suite.tx, qm.Where("email=?", o.User)).One()
 	suite.Equal(user.ID, op.UserID.Int64, "User")
 
-	// test with unknown user
```

Comment:
Im not sure about remove this test
is it ok that every one can send create operations now?